_posts/2024-11-18-geoserver-2-26-1-released.md

```diff
@@ -79,7 +79,7 @@ Community modules are shared as source code to encourage collaboration. If a top
 Additional information on GeoServer 2.26 series:
 
 * [GeoServer 2.26 User Manual](https://docs.geoserver.org/2.26.x/en/user/)
-* [GeoServer 2024 Q3 Developer Update]({% post_url 2024-07-22-developer-update.md %}) 
```

Comment:
@jodygarnett this addition to the series prevented the blog post deployment until I fixed it below